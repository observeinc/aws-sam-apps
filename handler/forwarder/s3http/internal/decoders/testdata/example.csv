a,b,c
1,2,
hello,world,nope
a,
