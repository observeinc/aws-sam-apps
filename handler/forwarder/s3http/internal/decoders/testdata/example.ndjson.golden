{"hello":"world"}
{"another":"value"}
{"and":3}
